wire-runtime/src/main/java/com/squareup/wire/FieldBinding.java

```diff
@@ -25,7 +25,7 @@
  * Read, write, and describe a tag within a message. This class knows how to assign fields to a
  * builder object, and how to extract values from a message object.
  */
-final class FieldBinding<M extends Message<M, B>, B extends Message.Builder<M, B>> {
+public final class FieldBinding<M extends Message<M, B>, B extends Message.Builder<M, B>> {
```

Comment:
Is exposed through `RuntimeMessageAdapter` hence should also become public

Reply:
Should also be in 'internal'

Reply:
That's in a different PR